src/components/Header.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <header>
+    <div class="header_block">TheBestCalculator</div>
+    <div class="header_block menu_block">
+      <ul class="menu">
+        <li class="active_link">
+          <a href="#" >Main</a>
+        </li>
+        <li>
+          <a href="#">History</a>
+        </li>
+      </ul>
+    </div>
+    <div class="header_block user_block" v-if="is_user_block_visible">
+      <div class="user_name">
+        Funny user
+        </div>
+      <div class="logout_element">
+      <a href="#"><i class="fas fa-power-off"></i> </a>
+      </div>
+    </div>
+  </header>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+        is_user_block_visible: true,
+    }
+  },
+  name: "HelloWorld",
+  props: {
+    msg: String
```

Comment:
This prop is unused, so please remove it.

Reply:
removed